packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -419,21 +419,21 @@ export class Copy {
             storclass: targetOptions.targetStorageClass,
             mgntclass: targetOptions.targetManagementClass,
             dataclass: targetOptions.targetDataClass,
-            dirblk: parseInt(dsInfo.dsorg == "PO" || dsInfo.dsorg == "POE"  ? "10" : "0")
+            dirblk: parseInt(dsInfo.dsorg == "PO" || dsInfo.dsorg == "PO-E"  ? "10" : "0")
         }));
     }
 
     /**
-     *  Private function to convert the ALC value from the format returned by the Get() call in to the format used by the Create() call
+     * Converts the ALC value from the format returned by the Get() call to the format used by the Create() call.
+     * @param {string} getValue - The ALC value from the Get() call.
+     * @returns {string} - The ALC value in the format used by the Create() call.
      */
-    private static convertAlcTozOSMF( zosmfValue: string): string {
-        /**
-         *  Create dataset only accepts tracks or cylinders as allocation units.
-         *  When the get() call retreives the dataset info, it will convert size
-         *  allocations of the other unit types in to tracks. So we will always
-         *  allocate the new target in tracks.
-        */
```

Comment:
Could we leave this comment here?
I guess I wouldn't want to loose some of the investigation that led to this comment 😋